### ISSUE TITLE
[REQUEST]Select first unwatched TV show Season/Episode

### ISSUE THREAD

**Issue by Dealz-416LS** [nixgates/plugin.video.seren#632]
**Is your feature request related to a problem? Please describe.**
The Kodi setting below doesn't work with Seren.

`Settings > Media Settings > Videos > (Library) Select first unwatched TV show season/episode`

**Describe the solution you'd like**
For this setting to work with Seren so that the first unwatched TV show season or episode is selected first.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by dplim87** [comment]
Seren, as of now anyways, is not integrated with Kodi's internal video library. The functionality you're looking for would be easy to implement yourself using a widget. Personally I create a widget that points to the Seren>My Shows>Next Up path so that it only lists the shows I've started watching. If you wanted to also include shows in your watchlist, create another widget pointing to My Shows>Watchlist. If you needed them to both be in the same widget an easy way to do so would be to use drinfernoo's AutoWidget plugin to create a widget with a static combined path. Integration of Seren into Kodi's internal library could be done, but I would honestly vote against it. I haven't found any plugin's library integration to work smoothly, so why complicate things to implement a "feature" that's already possible without it.

**Comment by drinfernoo** [comment]
@dplim87 That's not really what he's looking for.

The feature he's requesting is that if you enter into a show/season item, Kodi will automatically highlight the next season/episode of that show that you haven't watched yet.

**Comment by dplim87** [comment]
@drinfernoo 
Yes, but I had believed that would necessitate the items being in Kodi's media library, I could be wrong... I'd have to check out the relevant Kodi code. Perhaps, it is possible with plugin containers

**Comment by drinfernoo** [comment]
@dplim87 I believe you're correct. For that reason, it may not be possible, but we're leaving this open for now in hopes that it may be doable in the future.

**Comment by Dealz-416LS** [comment]
Not stale, active feature request.

**Comment by Dealz-416LS** [comment]
Not stale, active feature request.

**Comment by Dealz-416LS** [comment]
Not stale, active feature request.

**Comment by Dealz-416LS** [comment]
Not stale, active feature request.

**Comment by Dealz-416LS** [comment]
Not stale, active feature request.

**Comment by Dealz-416LS** [comment]
Not stale, active feature request.
